packages/humanize-distance/src/index.ts

```diff
@@ -1,32 +1,74 @@
+import { IntlShape } from "react-intl";
+
+function roundToOneDecimalPlace(number: number): number {
+  return Math.round(number * 10) / 10;
+}
+
 export function humanizeDistanceStringImperial(
   meters: number,
-  abbreviate?: boolean
+  abbreviate?: boolean,
+  intl?: IntlShape
 ): string {
   const feet = meters * 3.28084;
-  if (feet < 528)
-    return Math.round(feet) + (abbreviate === true ? " ft" : " feet");
-  return Math.round(feet / 528) / 10 + (abbreviate === true ? " mi" : " miles");
+
+  let unit;
+  let value;
+  let unitIfNoIntl;
+
+  if (feet < 528) {
+    unit = "foot";
+    unitIfNoIntl = abbreviate ? "ft" : "feet";
+    value = Math.round(feet);
+  } else {
+    unit = "mile";
+    unitIfNoIntl = abbreviate ? "mi" : "miles";
+    value = roundToOneDecimalPlace(feet / 5280);
```

Comment:
We could probably clean this up a bit if we used these values as the default case for those `let`s

Reply:
Sure, addressed in https://github.com/opentripplanner/otp-ui/pull/358/commits/9dcf6e7f50034fd0c9f980005e1637eb4e4c3796.